packages/manager/modules/vrack/src/vrack-services/ip/add/controller.js

```diff
@@ -0,0 +1,14 @@
+export default class IPv6SubnetAddModalCtrl {
+  onSubmit() {
+    if (this.formName.$valid) {
+      this.onConfirm()(this.subnet, this.address);
```

Comment:
you confuse `&` and `=` bindings. That's why you have a weird call functions here.
As you're using `&` bindings, you should set `data-on-confirm=myFunction(this.subnet, this.address)` directly in the modal html declaration